### PR TITLE
Sameer/admin_page

### PR DESCRIPTION
**Pull request by crazzywizard**
### Purpose
Create an admin page with login functionality using magic link

### CODE REVIEW

File: components/AdminPage/AdminPage.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+import { useUserProvider } from "../../providers/UserProvider"
+import Table, { StatusPill, SelectColumnFilter } from "./components/Table"
+
+type ITableDatum = {
+  walletAddress: string
+  tokenId: string
+  twitterHandle: string
+  reason: string
+  status: "Review" | "Accepted" | "Rejected"
+}
```
Comment (sweetmantech) [inline components/AdminPage/AdminPage.tsx:14]: nit: can we abstract all types to a specified `types` or `consts` file?

File: components/AdminPage/components/Table/Table.tsx [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+                <span className="sr-only">Last</span>
+                <ChevronDoubleRightIcon className="w-5 h-5" aria-hidden="true" />
+              </PageButton>
+            </nav>
+          </div>
+        </div>
+      </div>
+    </>
+  )
+}
```
Comment (sweetmantech) [inline components/AdminPage/components/Table/Table.tsx:90]: ooof this is a little large for me to read w/o getting confused.

any chance you can break this into sub-components?

**Review by sweetmantech — APPROVED** [review]
ship it 🚢

File: components/AdminPage/components/Table/Table.tsx (hunk 2 above)
Comment (crazzywizard) [reply components/AdminPage/components/Table/Table.tsx:90]: :done: broke it up as best as i could

**Pull request merged**